default.py

```diff
@@ -251,6 +344,60 @@ def set_input_take(self, arg):
             print(self.language('TEXT_COMMAND_USAGE'))
             sys.exit(2)
 
+    def show_available_settings(self):
+        """
+        Display valid settings and their aliases.
+        Prints the available settings along with their corresponding aliases.
+        """
+        print()
+        print('Valid settings:')
+        for aliases in self.config_mapping:
+            print(f"--setting {aliases[1]}=<value> ( alias: {aliases[0]} )")
+        print()
+
+    def set_setting(self, arg):
+        """
+        Set configuration settings based on user input.
+
+        Parses the input argument to determine the setting name and value.
+        If the input is not in the correct format, it displays available settings.
+        Otherwise, it sets the specified configuration value.
+
+        Args:
+            arg (str): Input argument in the format "<setting_name>=<value>".
+        """
+        pair = arg.split('=')
+        if len(pair) != 2:
+            self.show_available_settings()
+            return
+        name = pair[0].lower()
+        value = pair[1]
+
+        config_name = None
+
+        for aliases, setting_name in self.config_mapping.items():
+            if name in aliases:
+                config_name = setting_name
+                break
+
+        if config_name is None:
+            print('no matching configuration name for', name)
+            self.show_available_settings()
+            return
+
+        config_value = None
+        if value in ('true', 'True', 'yes', 'Y', 'y'):
+            config_value = True
+        elif value in ('false', 'False', 'no', 'N', 'n'):
+            config_value = False
+        elif value in ('none', 'None', 'NONE'):
+            config_value = None
+        else:
+            print('no valid configuration value for:', config_name)
```

Comment:
@cockroacher If I read this correctly this would not allow us to set string settings, for example:
`python default.py -i sites.json --setting browser=firefox`

Reply:
@cockroacher 
`python default.py -i sites.json --setting tests.sitespeed.iterations=5`
This doesn't work either I guess?
(Havn't tested)

Reply:
@cockroacher
could we make the default `config_value = True` (and allow for empty value)?
Then we could write stuff like:
`python default.py -i sites.json --setting improve-only`
instead of:
`python default.py -i sites.json --setting improve-only=true`

Reply:
Fixed, thanks :)